ros/src/force_torque_sensor_sim.cpp

```diff
@@ -65,7 +65,7 @@ ForceTorqueSensorSim::ForceTorqueSensorSim(ros::NodeHandle& nh) : nh_(nh), pub_p
     ftUpdateTimer_.start();
 }
 void ForceTorqueSensorSim::init_sensor() {
-    force_input_subscriber = nh_.subscribe("/base/twist_mux/command_teleop_joy", 1, &ForceTorqueSensorSim::subscribeData, this);
+    force_input_subscriber = nh_.subscribe("/cmd_vel", 1, &ForceTorqueSensorSim::subscribeData, this);
```

Comment:
Add here something like "cmd_force" and use remapping of parameters: http://wiki.ros.org/roslaunch/XML/remap